src/modules/swap/ModuleSwap.vue

```diff
@@ -905,6 +905,15 @@ export default {
       this.setTokenInValue(this.tokenInValue);
     },
     setTokenInValue: _.debounce(function (value) {
+      /**
+       * Ensure that both pairs have been set
+       * before calling the providers
+       */
+      const hasPair =
+        this.toTokenType.symbol &&
+        this.fromTokenType.symbol &&
+        !_.isEmpty(this.toTokenType) &&
+        !_.isEmpty(this.fromToTokenType);
```

Comment:
I think these two are redundant since if(this.toTokenType.symbol) tjem !_.isEmpty(this.toTokenType) is always true

Reply:
not necessarily, toTokenType is technically not empty when it's "Select Token" but is also not a valid to token